README.md

```diff
@@ -8,6 +8,22 @@ dependency management.
 ```sh
 pipenv sync # Installs all depedencies.
 ```
+# powershell
+```sh
+pipenv run python wap <args> <config file> -o <operation>
+```
+# example
+```sh
+pipenv run python wap -c config -o cancel
+```
+# bash
```

Comment:
Running the command in this manner is not `bash`-specific. `bash` is [a particular implementation](https://en.wikipedia.org/wiki/Bash_(Unix_shell)) of a Unix shell, but it is not the only one people use. There are many other shells, and a popular one you may run into is `zsh`.

Generally, when documenting usage information, you assume a POSIX shell. [POSIX](https://en.wikipedia.org/wiki/POSIX) is a set of a standards that compliant Unix systems adhere to (most Linux systems are _almost_ entirely compliant), to allow for code to be written in an interoperable way.

`bash` is not a POSIX standard. If you write `bash`-specific scripts, you have little confidence that it will work for all of your users. A POSIX shell, on the other hand, is something that essentially every Unix system is going to have, and is invoked as `sh`.

Reply:
I've changed bash to POSIX.. I've not pulled the standard to make sure the command will still work.